### DISCUSSION TITLE
Possibility to reorder scripts in head?

### DISCUSSION THREAD

**Discussion by niklaswolf** [unjs/unhead#118] [Q&A] [answered]
Is it possible currently to reorder the scripts in head at a later point?
My specific problem is, that I have to include a script as the first script in head, but i currently don't see a possibility to order the scripts? Maybe I'm missing something?

**Comment by harlan-zw** [comment, marked as answer]
> Is it possible currently to reorder the scripts in head at a later point? My specific problem is, that I have to include a script as the first script in head, but i currently don't see a possibility to order the scripts? Maybe I'm missing something?

Hey, did you see this page of the docs: https://unhead.harlanzw.com/guide/guides/sorting?

It might have some issues around hydration so let me know if you need further help

**Reply by niklaswolf** [reply]
Thanks so much, I don't know how I did not read this, it's clearly explained. My bad!
Sorry for the inconveniance :/

**Reply by niklaswolf** [reply]
Am I right that it's also possible to use the hook `tags:resolve` to do this?
I imagine a scenario where different plugins/components/... are adding a script with the same priority.

**Reply by harlan-zw** [reply]
Yup, you could use hooks as well.
